portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,72 @@
  */
 function addRandomFact() {
   const facts =
-      ['I have a pet dog', 'I like to code'];
+      ['I have a pet dog!', 'I like to code!', 'Marquees are my favourite HTML element!'];
 
   // Pick a random fact.
   const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
+}
+
+/**
+ * Controls slideshow image display on the page
+ */
+const decks = { // Enumerate slide decks
+  DOG_DECK: 0,
+  PLACE_DECK: 1,
+  FOOD_DECK: 2,
+};
+
+const SLIDES = [["/images/dog/dog0.jpg", "/images/dog/dog1.jpg", "/images/dog/dog2.jpg"],
+  ["/images/place/place0.jpg", "/images/place/place1.jpg", "/images/place/place2.jpg"],
+  ["/images/food/food0.jpg", "/images/food/food1.jpg", "/images/food/food2.jpg"]]
```

Comment:
Since you are only using these object values as indices for SLIDES, you can just merge there two into one.

```
const decks = { // Enumerate slide decks
  DOG_DECK: ["/images/dog/dog0.jpg", "/images/dog/dog1.jpg", "/images/dog/dog2.jpg"],
  PLACE_DECK: ["/images/place/place0.jpg", "/images/place/place1.jpg", "/images/place/place2.jpg"],
  FOOD_DECK: ["/images/food/food0.jpg", "/images/food/food1.jpg", "/images/food/food2.jpg"],
};

Reply:
Responding to your comment on chat

I see I thought object keys when they aren't known types like what you did, would automatically convert to some kind of enum int. What I found hard to read was not the enum but the 2D array. Hard to tell from the initializer what it is. So keep the enum but change the 2D array to be an object.

const slides = { 
  decks.DOG_DECK: ["/images/dog/dog0.jpg", "/images/dog/dog1.jpg", "/images/dog/dog2.jpg"],
  decks.PLACE_DECK: ["/images/place/place0.jpg", "/images/place/place1.jpg", "/images/place/place2.jpg"],
  decks.FOOD_DECK: ["/images/food/food0.jpg", "/images/food/food1.jpg", "/images/food/food2.jpg"],
};